exercises/concept/chrono-realms-time-tree/.meta/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "<your_gh_username>",
+    "0xNeshi"
+  ],
+  "files": {
+    "solution": [
+      "src/lib.cairo"
+    ],
+    "test": [
+      "tests/chrono_realms_time_tree.cairo"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "blurb": "<blurb>"
```

Comment:
The blurb is required

Reply:
See https://github.com/exercism/cairo/pull/272#discussion_r1777017124